src/pretix/base/services/vouchers.py

```diff
@@ -44,3 +45,36 @@ def vouchers_send(event: Event, vouchers: list, subject: str, message: str, reci
                         'message': message,
                     }
                 )
+
+@app.task(base=TransactionAwareProfiledEventTask, acks_late=True)
+def vouchers_resend(event: Event, vouchers: list, user: int) -> None:
+    vouchers = list(Voucher.objects.filter(id__in=vouchers).order_by('id'))
+    user = User.objects.get(pk=user)
+    for v in vouchers:
+        with language(event.settings.locale):
+            recipient = v.recipient
+            # FIXME We can't get the original name
+            name = ''
+            # FIXME We can't get the original subject
+            subject = gettext('Resent Voucher')
+            # FIXME We can't get the original message
+            message = LazyI18nString('We are resending you your voucher for {event}: {voucher_list}')
```

Comment:
This doesn't get translated this way, does it?
Should I do this?:
```suggestion
            message = LazyI18nString(_('We are resending you your voucher for {event}: {voucher_list}'))
```

Reply:
No, the correct way is using LazyI18nString.from_gettext, see for example pretix/base/settings.py for usage